### DISCUSSION TITLE
collection.flush() getting failed

### DISCUSSION THREAD

**Discussion by navneetrajput06** [milvus-io/milvus#35364] [Q&A and General discussion]
Hi team,
Could you please help me in resolve below issue.
>> collection.flush()
E0807 17:41:46.652831000 6266368000 hpack_parser.cc:999]               Error parsing 'content-type' metadata: invalid value
E0807 17:42:46.733381000 6266368000 hpack_parser.cc:999]               Error parsing 'content-type' metadata: invalid value
E0807 17:43:46.838769000 6266368000 hpack_parser.cc:999]               Error parsing 'content-type' metadata: invalid value

[4:51](https://a1345092.slack.com/archives/D017WJ542Q5/p1723074704202709)

**Comment by yhmo** [comment]
Similar issues:
https://github.com/milvus-io/milvus/issues/29096

It is a grpc level error, maybe related to reverse proxy:
https://github.com/grpc/grpc/issues/29706
